datalad/interface/common_opts.py

```diff
@@ -33,6 +33,13 @@
     constraints=EnsureInt() | EnsureNone(),
     doc="""limit recursion into subdataset to the given number of levels""")
 
+shared_access_opt = Parameter(
+    args=('--shared-access',),
```

Comment:
Well... I don't see how they ever really need to be different, since in both cases we are taking about "create", just that here it is local and the other one on a remote location.